styleguide/source/_patterns/01-molecules/05-navigation/00-primary-nav/primary-nav.twig

```diff
@@ -1,134 +1,124 @@
-<div class="nav-primary-menu_button">Menu</div>
+<div class="nav-primary_button">Menu</div>
 
-<nav id="nav-primary" class="nav-primary-menu" role="navigation">
+<nav class="nav-primary" role="navigation" aria-label="Main Navigation">
 
-  <ul class="nav-primary_list nav-primary_list-closed" role="menubar">
-    <li class="nav-primary_list-item">
-      <div class="nav-primary_list-item_title">
-        {% include '04-link-primary-nav.twig' with { 'content': nav.first, 'url': '/patterns/04-pages-00-primary-landing-page-education/04-pages-00-primary-landing-page-education.html' } %}
-      </div>
+  <ul class="nav-primary_list nav-primary_list-mobile-closed" role="menubar">
+    <li class="nav-primary_section">
+        {% include '04-link-primary-nav.twig' with { 'content': nav.first } %}
```

Comment:
I am still seeing this issue on desktop. Not sure right now why it's happening, but if you're burned out on this and want me to take a closer look, I will.

![primary_nav_llinks_linking](https://user-images.githubusercontent.com/28711067/26898835-d97872b0-4b92-11e7-8b12-90661c5ef484.gif)